sbncode/FlashMatch/FlashPredict_module.cc

```diff
@@ -20,6 +20,7 @@ FlashPredict::FlashPredict(fhicl::ParameterSet const& p)
   , fOpHitARAProducer(p.get<std::string>("OpHitARAProducer", ""))
   , fOpFlashProducer(p.get<std::vector<std::string>>("OpFlashProducer", {}))
   , fOpFlashHitProducer(p.get<std::vector<std::string>>("OpFlashHitProducer", {}))
+  , fVerbose(p.get<bool>("Verbose", false))
```

Comment:
A little note for future things. It's often good not to define defaults unless totally necessary. Makes fcl debugging harder :smile: